applications/ChimeraApplication/custom_processes/calculate_signed_distance_to_2d_condition_skin_process.h

```diff
@@ -0,0 +1,2682 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+// ==============================================================================
+//  ChimeraApplication
+//
+//  License:         BSD License
+//                   license: ChimeraApplication/license.txt
+//
+//  Authors:        Aditya Ghantasala, https://github.com/adityaghantasala
+// 					Navaneeth K Narayanan
+//					Rishith Ellath Meethal
+// ==============================================================================
+//
+#if !defined(KRATOS_CALCULATE_SIGNED_DISTANCE_CONDITION_2D_PROCESS_H_INCLUDED)
+#define KRATOS_CALCULATE_SIGNED_DISTANCE_CONDITION_2D_PROCESS_H_INCLUDED
```

Comment:
This file is MASSIVE and appears to be largely duplication of https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/processes/calculate_signed_distance_to_3d_condition_skin_process.h. 

This is not maintainable.

Reply:
@msandre  agreed. That is the reason I want to completely remove this. I am now experimenting with the new distance calculation processes from the core (CalculateDistanceToSkinProcess) . I mentioned this in the initial comments.

Reply:
ok, sorry, I missed that part. Perhaps you can put a comment block at the top of the header like

```
THIS IS A TEMPORARY FILE. DO NOT SPEND TIME REVIEWING OR FIXING IT!!!!!!!!!!!!!!!!!!!!
```
Once such a file gets into a PR or the master, there is the risk that some wastes a considerable amount of time trying to clean it up or review it.

Reply:
@msandre now this file is removed !